envoy/config/filter/http/transcoder/v2/transcoder.proto

```diff
@@ -55,4 +55,16 @@ message GrpcJsonTranscoder {
   // `JsonPrintOptions <https://developers.google.com/protocol-buffers/docs/reference/cpp/
   // google.protobuf.util.json_util#JsonPrintOptions>`_.
   PrintOptions print_options = 3;
+
+
+  // Whether to skip recalculating the route after the
```

Comment:
"Whether to keep the incoming request route"? To be precise, recalculation doesn't happen with in the filter, it is calculated when it is necessary.

Also please reflow the comment, the column limit is 100.

Reply:
afaict there are no columns past 100 (max is 83 that i see)
can you show me the line that is over?

Reply:
:+1: